poznaj/stories/tests/test_views.py

```diff
@@ -76,3 +79,45 @@ def test_update_story(self):
         self.assertEqual(database_story.title, 'new_title')
         self.assertEqual(database_story.description, 'new_description')
         self.assertEqual(str(database_story.duration), '1:00:00')
+
+    def test_filter_story(self):
```

Comment:
hej @robert-skarzycki dzięki za komentarz! Super pomocny. Obecnie działa to tak że sortuje po odległości od pierwszego punktu. Rozumiem że zastanawiasz się czy nie zrobić tego na 2 razy:

1. obliczenie odległości
2. sortowanie

Jeśli tak to nie wiem czy nie było by to testowanie trochę jak działa GeoDjango. 

Co do assercji to masz rację - nie ma potrzeby porównywania słowników